### ISSUE TITLE
Object required when using `version` parameter on `get` resource

### ISSUE THREAD

**Issue by psellars-hyprnz** [concourse/github-release-resource#91]
I am just trying to get some clarity around this section of the documentation and what I have experienced:

https://github.com/concourse/github-release-resource#check-check-for-released-versions

With the following code:

```
- get: my-release
    version:
      tag: X.X,X
```

I was seeing a `pinned version {"tag":"X.X.X"} is unavailable.

I had done a `fly -t myteam check-resource -r pipeline/my-release -f X.X.X`

The documentation states the need for an object to be passed when using the `version` parameter. So I updated the above to:

```
- get: my-release
    version:
      tag: X.X,X
      id: "XXXXXXXX"
      timestamp: "2019-08-30T12:51:57Z"
```

..and the check was successful.

Am just looking to clarify that this is the requirement, as when I used just the tag and id it didn't work either. The complete object needs to be passed in here?

For clarity may be worth adding an example like this to the README if that is the case.

Thanks for the excellent work, hopefully clarity can be achieved :)

**Comment by Freakin** [comment]
Having the same issue here.  Did you ever figure out a solution besides passing in all the properties from the resource version
 @psellars-hyprnz ?

**Comment by silvamerica** [comment]
I'm experiencing the same issue, and it's quite frustrating. Having to pass in the ID, specifically, makes this as a terrible experience, as the ID is only determined after the resource has been checked and can be changed if something happens to the resource (like a tag filter gets misapplied and then un-done).

**Comment by silvamerica** [comment]
It's been confirmed as a bug and an issue has been opened! https://github.com/concourse/concourse/issues/5535

**Comment by aoldershaw** [comment]
Closing this as concourse/concourse#5535 has been fixed by concourse/concourse#5780
